### ISSUE TITLE
NiceToHave v1.1.0 Client Crash

### ISSUE THREAD

**Issue by AkipTsaqif** [RedstoneParadox/NiceToHave#7]
The newest update of NiceToHave still crashes my game on startup. This crash keeps happening with every version starting from 1.0.2.

[crash-2019-08-10_17.19.33-client.txt](https://github.com/RedstoneParadox/NiceToHave/files/3488747/crash-2019-08-10_17.19.33-client.txt)

```
---- Minecraft Crash Report ----
// I feel sad now :(

Time: 8/10/19 5:19 PM
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors!
	at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:45)
	at net.fabricmc.loader.entrypoint.hooks.EntrypointClient.start(EntrypointClient.java:34)
	at net.minecraft.class_310.method_1503(class_310.java)
	at net.minecraft.class_310.method_1514(class_310.java:394)
	at net.minecraft.client.main.Main.main(Main.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:170)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:129)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	Suppressed: java.lang.NullPointerException
		at net.minecraft.class_2302.getSeed(class_2302.java:1522)
		at net.redstoneparadox.nicetohave.misc.DispenserBehaviors.blockToDispenserBehavior(DispenserBehaviors.kt:63)
		at net.redstoneparadox.nicetohave.misc.Listeners$initListeners$2.onEntryAdded(Listeners.kt:35)
		at net.redstoneparadox.nicetohave.misc.Listeners$initListeners$2.onEntryAdded(Listeners.kt:21)
		at net.minecraft.class_2370.lambda$null$0(class_2370.java:563)
		at net.minecraft.class_2370.handler$setPost$zff000(class_2370.java:645)
		at net.minecraft.class_2370.method_10273(class_2370.java:45)
		at net.minecraft.class_2348.method_10273(class_2348.java:24)
		at net.minecraft.class_2370.method_10272(class_2370.java:50)
		at net.minecraft.class_2378.method_10230(class_2378.java:183)
		at com.elytradev.engination.block.EnginationBlocks.init(EnginationBlocks.java:196)
		at com.elytradev.engination.Engination.onInitialize(Engination.java:34)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.lambda$logErrors$0(EntrypointUtils.java:38)
		at java.util.ArrayList.forEach(Unknown Source)
		at net.fabricmc.loader.entrypoint.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:36)
		... 11 more
```

**Comment by RedstoneParadox** [comment]
Please fill out the entire issue template [issue template](https://github.com/RedstoneParadox/NiceToHave/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) so that I have all the information needed to fix this issue.

**Comment by AkipTsaqif** [comment]
**Nice to Have version:** 1.1.0

**Fabric Loader Version:** 0.4.8 build 159 1.14.4

**Fabric API Version:** 0.3.1 build 138

**Mappings version:** 1.14.4 build 9

**Fabric Language Kotlin Version:** 1.3.40 build 1

**Name and version of any other mods present:** already attached in the error logs above

**What actually happened:** the game just crash whenever I press start on the launcher

**Steps to reproduce:** install everything and then launch the game

I could upload the zip for all the other mods present if you want

**Comment by RedstoneParadox** [comment]
Okay, so the crash report had enough information to go by and I was able to fix it for version 1.1.1.

**Comment by AkipTsaqif** [comment]
Alright thanks. It worked now
